Install/build.bat

```diff
@@ -0,0 +1,31 @@
+@echo off
+
+set PATH=C:\Program Files (x86)\Inno Setup 6
+
+:check_win32
```

Comment:
These `goto` statements and labels are hard to read. Isn't it possible to simply use `if` statements followed by a block of code, such as:

```bat
if exist "..\Win32\Release\X584.exe" (
    echo Building installer for X584 x86 version...
    ISCC /DX584_PLATFORM=Win32 x584.iss
)
```